### ISSUE TITLE
You have to manually load the cucumber-tag-expressions main file instead of simply requiring the npm module.

### ISSUE THREAD

**Issue by jdrew1303** [cucumber/cucumber#104]
## Summary
At the moment if you install the cucumber-tag-expressions npm module and want to use it in your code you have to call it like so: 

```javascript
const TagExpressionParser = require('cucumber-tag-expressions/lib/tag_expression_parser');
```

A `main` property pointing to the file in the `package.json` would allow you to do a straight `require('cucumber-tag-expressions')`.

## Expected Behavior
You should be able to `require('cucumber-tag-expressions')` and have access to the library without having to traverse to the lib folder.

## Current Behavior
You have to traverse to the lib folder.

## Possible Solution
A `main` property pointing to the file in the `package.json`.

## Steps to Reproduce (for bugs)
1. Try loading it directly in any project.

## Context & Motivation
I'm using the tag parser as a simple method to filter documents by giving a user an easy dsl for working with document labels. Im also using it with angular (1.6) to allow me to find modules that rely on a certain group of dependencies (angular stores the dependency information as an array of strings). This library is actually really useful in a wide range of scenarios, outside of cucumber ones (pun intended 😁 ).

## Your Environment
Environment independent

**Comment by charlierudolph** [comment]
I believe this was fixed in #76 and a new version just needs to be released. @aslakhellesoy any chance this tag-expressions-javascript could get a new release?

**Comment by jdrew1303** [comment]
@charlierudolph Ah ok fair enough.
